app/routes/wallet/jws.py

```diff
@@ -64,15 +85,31 @@ async def sign_jws(
     "/verify",
     response_model=JWSVerifyResponse,
     summary="Verify JWS",
-    description="""
-Verify JSON Web Signature (JWS)
-
-See https://www.rfc-editor.org/rfc/rfc7515.html for the JWS spec.""",
 )
 async def verify_jws(
     body: JWSVerifyRequest,
     auth: AcaPyAuth = Depends(acapy_auth_from_header),
 ) -> JWSVerifyResponse:
+    """
+    Verify JSON Web Signature (JWS)
+    ---
+
+    See https://www.rfc-editor.org/rfc/rfc7515.html for the JWS spec.
+
+    Request body:
+    ---
+        JWSVerifyRequest: The JWS to verify.
+            jws: str
+
+    Returns:
+    ---
+        JWSVerifyResponse
+            payload: dict: The payload of the JWS.
+            headers: dict: The headers of the JWS.
+            kid: str: The key id of the signer.
+            valid: bool: Whether the JWS is valid.
+            error: str: Error text.
```

Comment:
Also these descriptions should be more informative. "Error text."? " A description if a verification error occurred"